server/public/styles.css

```diff
@@ -226,7 +227,7 @@ h4 {
   position: relative;
 }
 
-.episodes-list > img {
+.episodes-list-image {
```

Comment:
You should set a `height` on this too so that all of your images are the exact same size and aspect ratio. You'll need `object-fit: contain` as well, to prevent the images from getting stretched or squeezed looking.

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit